src/compas_fab/robots/robot.py

```diff
@@ -496,22 +549,30 @@ def get_group_configuration(self, group, full_configuration):
     def merge_group_with_full_configuration(self, group_configuration, full_configuration, group):
         """Returns a robot's full configuration by merging a group's configuration with a full configuration.
 
+        If the configuration of the joints differ, the group configuration takes
```

Comment:
`configuration of the joints` -> that feels redundant, configuration is always "of joints", but more importantly, the sentence is unclear to me: "If the [config] differ, ..." -> what exactly differs from what? I guess you meant, something like "if the number of joints in the full configuration differs from that of the group configuration [...]"

Reply:
First of, good point.

My reasoning for adding that was to make it more clear that while merging the group conf overrides the full conf for joints present in both group and full conf.

Reply:
But in that case, make it even more explicit: `The group configuration takes precedence over the full configuration in case a joint value is present in both.`

Reply:
71d5122